content/en/docs/concepts/security/hardening-guide/authorization-hardening.md

```diff
@@ -0,0 +1,89 @@
+---
+title: Hardening Guide - Authorization 
+description: >
+  Information on authorization options in Kubernetes and their security properties.
+content_type: concept
+weight: 90
+---
+
+<!-- overview -->
+
+Authorization in Kubernetes ecosystem means specifying the privileges/permissions for a particular kubernetes resource. Defining a proper
+authorization policy is an important step in preventing unauthorized access to the clusters or privilege escalation attacks. Kubernetes offers
```

Comment:
If this sentence is referring to a single cluster then I suggest the following
```suggestion
authorization policy is an important step in preventing unauthorized access to the cluster or privilege escalation attacks. Kubernetes offers
```

If this sentence is referring to multiple clusters then I suggest
```suggestion 
authorization policy is an important step in preventing unauthorized access to clusters or privilege escalation attacks. Kubernetes offers
```